2.0/flask_app/services/llm_service.py

```
@@ -0,0 +1,64 @@
+        logging.info(f'Querying ollama with prompt: {prompt}\nContext: {context}')
+        try:
+            response = requests.post(
+                f"{self.ollama_url}/api/generate",
+                json={
+                    "model": self.ollama_model,
+                    "prompt": f"Use the following SEC filing excerpts to answer the user's question\n\n: {context}\n\nQuestion: {prompt}\nAnswer:",
+                    "stream": False
+                },
+                timeout=300
+            )
+            logging.info(f'Sent ollama query.')
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)